src/Data/String/Regex.purs

```diff
@@ -93,18 +93,25 @@ foreign import _match
 match :: Regex -> String -> Maybe (NonEmptyArray (Maybe String))
 match = _match Just Nothing
 
--- | Replaces occurences of the `Regex` with the first string. The replacement
+-- | Replaces occurrences of the `Regex` with the first string. The replacement
 -- | string can include special replacement patterns escaped with `"$"`.
 -- | See [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace).
 foreign import replace :: Regex -> String -> String -> String
 
--- | Transforms occurences of the `Regex` using a function of the matched
--- | substring and a list of submatch strings.
--- | See the [reference](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace#Specifying_a_function_as_a_parameter).
-replace' :: Regex -> (String -> Array String -> String) -> String -> String
-replace' = replaceBy
+foreign import _replaceBy
+  :: (forall r. r -> Maybe r)
+  -> (forall r. Maybe r)
+  -> Regex
+  -> (String -> Array (Maybe String) -> String)
+  -> String
+  -> String
```

Comment:
I moved this import above the line that references it, for consistency with the placement of `_match` and `_search`.